armi/physics/neutronics/fissionProductModel/fissionProductModel.py

```diff
@@ -152,44 +148,61 @@ def _fissionProductBlockType(self):
 
     def interactBOL(self):
         interfaces.Interface.interactBOL(self)
-        self.setAllBlockLFPs()
+        if self._explicitFissionProducts:
+            self.setAllComponentFissionProducts()
+        else:
+            self.setAllBlockLFPs()
+
+    def setAllComponentFissionProducts(self):
+        """
+        Initialize all blueprint nuclides for each ``DEPLETABLE`` component in the core.
+
+        Notes
+        -----
+        This should be called when explicit fission product modeling is enabled to
+        ensure that fission products are initialized on the depletable components within
+        the reactor data model.
+
+        .. note::
```

Comment:
You have a `.. note::` within a `Notes`? 

I would ditch the `.. note::`, since you already have a section for notes. IMO.